internal/auth/internal/types/computed_traffic_permissions.go

```diff
@@ -67,5 +67,6 @@ func aclReadHookComputedTrafficPermissions(authorizer acl.Authorizer, authzConte
 }
 
 func aclWriteHookComputedTrafficPermissions(authorizer acl.Authorizer, authzContext *acl.AuthorizerContext, res *pbresource.Resource) error {
-	return authorizer.ToAllowAuthorizer().TrafficPermissionsWriteAllowed(res.Id.Name, authzContext)
+	// users should not be writing computed resources
+	return authorizer.ToAllowAuthorizer().OperatorWriteAllowed(authzContext)
```

Comment:
Test failing on ACL changes. Will fix

Reply:
There is another PR refactoring ACL tests that hasn't been merged yet. Will remove ACLs from this PR until that other PR gets merged